ProjectSettings/QualitySettings.asset

```diff
@@ -4,7 +4,7 @@
 QualitySettings:
   m_ObjectHideFlags: 0
   serializedVersion: 5
-  m_CurrentQuality: 5
+  m_CurrentQuality: 0
```

Comment:
Was this change to quality settings intentional?

Reply:
no it wasnt.

Reply:
changed back to 5